ref_ts/tests/test_ref_ts.py

```diff
@@ -0,0 +1,239 @@
+# -*- coding: utf-8 -*-
+import json
+import logging
+
+from django.test import TestCase, Client
+from django.contrib.auth.models import User, Group
+
+from hs_core import hydroshare
+from ref_ts.models import RefTimeSeriesResource
+import ref_ts.views as views
+
+
+class TestRefTS(TestCase):
+
+    def setUp(self):
+        self.logger = logging.getLogger("ref_ts")
+        self.api_client = Client()
+
+        self.username = 'creator'
+        self.password = 'mybadpassword'
+
+        self.group, _ = Group.objects.get_or_create(name='Hydroshare Author')
+
+        # create a user to be used for creating the resource
+        self.user = hydroshare.create_account(
+            'creator@hydroshare.org',
+            username=self.username,
+            first_name='Creator_FirstName',
+            last_name='Creator_LastName',
+            superuser=False,
+            password=self.password,
+            groups=[self.group]
+        )
+
+        self.url_to_get_his_urls = "/hsapi/_internal/get-his-urls/"
+        self.url_to_search_sites = "/hsapi/_internal/search-sites/"
+        self.url_to_search_variables = "/hsapi/_internal/search-variables/"
+        self.url_to_time_series_from_service = "/hsapi/_internal/time-series-from-service/"
+        self.url_to_create_ref_time_series = "/hsapi/_internal/create-ref-time-series/"
+        self.url_to_download_resource_files = "/hsapi/_internal/{0}/download-files/"
```

Comment:
Can you use /hsapi/resource/<pk>/file/<filename>/ for this?  If so, then we should get rid of this view as it duplicates what is in the REST API.